README.md

```diff
@@ -127,11 +127,11 @@ and the remaining maps *must* have the same set of keys and values:
              {:name "Andrew" :surname "Cooper" :age 12}
              {:name "Jane" :surname "Daniels" :age 56}])
     sql/format)
-=> ["INSERT INTO properties (age, name, surname) 
-     VALUES (34, ?, ?), (12, ?, ?), (56, ?, ?)" 
-    "John" "Smith" 
-    "Andrew" "Cooper" 
-    "Jane" "Daniels"]
+=> ["INSERT INTO properties (name, surname, age)
```

Comment:
This change is probably Clojure, not honeysql -- we're looking at map keys, so Clojure gets to decide in what order to iterate over them.